jobs/topline_dashboard.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -x
+
+if [[ -z "$mode" ]]; then
+   echo "Missing arguments!" 1>&2
+   exit 1
+fi
+
+bucket="net-mozaws-prod-metrics-data"
```

Comment:
Yep, this should be handled in the corresponding code.

https://github.com/mozilla/python_mozetl/blob/master/mozetl/topline/topline_dashboard.py#L140